lib/reciter.rb

```diff
@@ -1,52 +1,67 @@
 class Reciter
 
   def recite_rhyme
-    "This is the house that Jack built.\n" +
-    "This is the malt that lay in the house that Jack built.\n" +
-    "This is the rat that ate the malt that lay in the house that Jack built.\n" +
-    "This is the cat that killed the rat that ate the malt that lay in the house that Jack built.\n" +
-    "This is the dog that worried the cat that killed the rat that ate the malt that lay in the house that Jack built.\n" +
-    "This is the cow with the crumpled horn that tossed the dog that worried the cat that killed the rat that ate the malt that lay in the house that Jack built.\n" +
-    "This is the maiden all forlorn that milked the cow with the crumpled horn that tossed the dog that worried the cat that killed the rat that ate the malt that lay in the house that Jack built.\n" +
-    "This is the man all tattered and torn that kissed the maiden all forlorn that milked the cow with the crumpled horn that tossed the dog that worried the cat that killed the rat that ate the malt that lay in the house that Jack built.\n" +
-    "This is the priest all shaven and shorn that married the man all tattered and torn that kissed the maiden all forlorn that milked the cow with the crumpled horn that tossed the dog that worried the cat that killed the rat that ate the malt that lay in the house that Jack built.\n" +
-    "This is the rooster that crowed in the morn that woke the priest all shaven and shorn that married the man all tattered and torn that kissed the maiden all forlorn that milked the cow with the crumpled horn that tossed the dog that worried the cat that killed the rat that ate the malt that lay in the house that Jack built.\n" +
-    "This is the farmer sowing his corn that kept the rooster that crowed in the morn that woke the priest all shaven and shorn that married the man all tattered and torn that kissed the maiden all forlorn that milked the cow with the crumpled horn that tossed the dog that worried the cat that killed the rat that ate the malt that lay in the house that Jack built.\n" +
-    "This is the horse and the hound and the horn that belonged to the farmer sowing his corn that kept the rooster that crowed in the morn that woke the priest all shaven and shorn that married the man all tattered and torn that kissed the maiden all forlorn that milked the cow with the crumpled horn that tossed the dog that worried the cat that killed the rat that ate the malt that lay in the house that Jack built.\n"
+    index = 0
+    continous_phrase = ""
+    rhyme = ""
+
+    while index < 12
+      continous_phrase = phrases[index].strip + " " + continous_phrase
```

Comment:
Thought: I think this is fine, but "set an empty variable and fill it up" in ruby is usually a code smell. I wonder if there's a way to do this via `#map` or even extract a class somewhere?